docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -1,122 +1,44 @@
 ---
 title: "Verify cluster"
 linkTitle: "Verify cluster"
-weight: 11
+weight: 3
 aliases:
     /docs/tasks/cluster/cluster-verify/
 date: 2017-01-05
 description: >
   How to verify an EKS Anywhere cluster is running properly
 ---
 
-**Verify an EKS Anywhere cluster from the target cluster**
-First, set the `KUBECONFIG` environment variable is set up for the target cluster (management or workload cluster) of choice:
+### Verify an EKS Anywhere cluster
+
+First, set the `KUBECONFIG` environment variable for your cluster:
```

Comment:
Which kubeconfig do I use? Does it change if I am running a standalone cluster vs separate management / workload clusters? Lets clarify that